_episodes/documentation.md

```diff
@@ -1,33 +1,92 @@
+questions about the project: why were these data collected, by whom, and what
+questions do they address? Was there any kind of quality assurrence?
 
-Project information: how was the data obtained (where there steps)? By whom? Are there different versions of the data? Quality assurance?
+## Where to document my data?
 
-On data specific info you often find difficult terms like ‘semantics’, ‘vocabularies’, ‘ontologies’, ‘linked data’. In climate (related) science, these are often so ‘logical’ that you won’t even notice they’re there. However, look at this example of a patient’s data model, and the netcdf model. Do you see it now? More info on this in sections file format and metadata.
+Documentation can take many forms: a data management plan, papers published,
+(jupyter) notebooks, lab journals, provenance information, etc. It is a good
+idea to bundle these with the data when you publish the data, so that people
+that find your data will not have to search for it elsewhere. Most data
+repositories have a 'description' field, which is a suitable place to provide
+any information that is not covered by the other metadata fields.
 
-Project info is typically written in the data description fields, but may also include pointers to external documents, such as the data management plans.
+> ## Grouping related files with Zenodo communities
+> Data documentation in your project may be scattered over many files of
+> different types. To bundle this information, Zenodo provides 'communities'.
+> You can upload all your files to Zenodo and add them to a community. You can
+> also track versions. Each file gets its own DOI. So you could e.g. cite your
+> data management plan in the description field of your dataset.
+{: .callout}
 
-Point forward to next episodes (e.g. formats and metadata help to document data).
+## What is the difference between metadata and documentation?
 
-Data management plan describes the planning/methods of data collection activities, data processing, analysis and (long-term) preservation.
+Following <https://howtofair.dk>, we discuss data documentation and metadata
+separately. However, the distinction between the two is not always clear. In
+this episode, we will focus on the rich, descriptive kind of documentation
+that is mostly relevant for human interpretation. In the next episodes, we will
+focus on more formalized metadata that is also, or even mainly, intended for
+machine readability.
 
-You can also log other important decisions that impact your data (sort of labjournal?!).
+## Exercises
 
-Documentation can take many forms: data management plan, papers published, (jupyter) notebooks, provencance information, etc. It is a good idea to bundle these with the data when you publish the data, so that people that find your data will not have to search for it elsewhere
+> ## Checking data documentation
+>
+> Visit a data repository of your choosing (e.g. hydroshare, 4TU, Zenodo, ...).
```

Comment:
Should we include links to these repositories or is this part of the fun? ;-)

Reply:
part of the fun I would say.